templates/email/default/problem-confirm.html

```diff
@@ -18,7 +18,7 @@ <h1 style="[% h1_style %]">Please confirm your&nbsp;report</h1>
 of problem, so it will instead be sent to [% report.body %].
 [% END %]
 
-[% TRY %][% INCLUDE '_problem-confirm_extra.html' %][% CATCH file %][% END %]
+[% TRY %][% INCLUDE '_problem-confirm_extra.html' report=report %][% CATCH file %][% END %]
```

Comment:
I don't think this is necessary; variables are already available to included templates, you only need this if you need to pass in something under a different name.